index.bs

```diff
@@ -446,6 +446,359 @@ A file containing a 'pict' track compliant with this profile is expected to list
+  <table class="boxlist">
+        <tr>
+            <th>Level 1</th>
+            <th>Level 2</th>
+            <th>Version(s)</th>
+            <th>Specification</th>
+            <th>Containing Box</th>
+            <th>Description</th>
+        </tr>
+        <tr>
+            <td>dinf</td>
+            <td>&nbsp;</td>
+            <td>-</td>
+            <td>ISOBMFF</td>
+            <td>meta</td>
+            <td>&nbsp;</td>
+        </tr>
+        <tr>
+            <td>&nbsp;</td>
+            <td>dref</td>
+            <td>0</td>
+            <td>ISOBMFF</td>
+            <td>&nbsp;</td>
+            <td>&nbsp;</td>
+        </tr>
+        <tr>
+            <td>iref</td>
+            <td>&nbsp;</td>
+            <td>0, 1</td>
+            <td>ISOBMFF</td>
+            <td>meta</td>
+            <td>Used to indicate directional relationships between images or metadata</td>
+        </tr>
+        <tr>
+            <td>&nbsp;</td>
+            <td>auxl</td>
+            <td>-</td>
+            <td>HEIF</td>
+            <td>&nbsp;</td>
+            <td>Used when an image is auxiliary to another image</td>
+        </tr>
+        <tr>
+            <td>&nbsp;</td>
+            <td>thmb</td>
+            <td>-</td>
+            <td>HEIF</td>
+            <td>&nbsp;</td>
+            <td>Used when an image is a thumbnail of another image</td>
+        </tr>
+        <tr>
+            <td>&nbsp;</td>
+            <td>dimg</td>
+            <td>-</td>
+            <td>HEIF</td>
+            <td>&nbsp;</td>
+            <td>Used when an image is derived from another image</td>
+        </tr>
+        <tr>
+            <td>&nbsp;</td>
+            <td>prem</td>
+            <td>-</td>
+            <td>MIAF</td>
+            <td>&nbsp;</td>
+            <td>Used when when an alpha image contains premultiplied color values from another image</td>
+        </tr>
+        <tr>
+            <td>&nbsp;</td>
+            <td>cdsc</td>
+            <td>-</td>
+            <td>HEIF</td>
+            <td>&nbsp;</td>
+            <td>Used to link metadata with an image</td>
+        </tr>
+        <tr>
+            <td>idat</td>
+            <td>&nbsp;</td>
+            <td>-</td>
+            <td>ISOBMFF</td>
+            <td>meta</td>
+            <td>Used to store derived image definitions</td>
+        </tr>
+        <tr>
+            <td>grpl</td>
+            <td>&nbsp;</td>
+            <td>-</td>
+            <td>ISOBMFF</td>
+            <td>meta</td>
+            <td>Used to indicate that multiple images are semantically grouped</td>
+        </tr>
+        <tr>
+            <td>&nbsp;</td>
+            <td>altr</td>
+            <td>-</td>
```

Comment:
ISOBMFF lists `altr` as being a full box. This should probably be `0`.

Reply:
Good catch. This is related to this issue https://github.com/MPEGGroup/FileFormat/issues/44

Reply:
Fixed